Remotion/ObjectBinding/Web.ClientScript/Scripts/BocList.ts

```diff
@@ -0,0 +1,716 @@
+
+    if (hasScrollMoveFromScrollbar)
+      fakeTableHeadContainer.scrollLeft = scrollLeft;
+  }
+
+  public static InitializeNavigationBlock (pageNumberField: JQuery, pageIndexField: JQuery): void
+  {
+    ArgumentUtility.CheckNotNullAndTypeIsObject ('pageNumberField', pageNumberField);
+    ArgumentUtility.CheckNotNullAndTypeIsObject('pageIndexField', pageIndexField);
+  
+    pageNumberField.bind('change', function () {
+  
+      var pageNumber = parseInt(pageNumberField.val(), 10);
+      if (isNaN (pageNumber) || !TypeUtility.IsInteger (pageNumber))
+      {
+        if (pageNumberField.val().length > 0)
+          setTimeout(function () { pageNumberField.focus(); }, 0);
+  
+        pageNumberField.val("" + (parseInt(pageIndexField.val(), 10) + 1));
+        return false;
+      }
+      else
+      {
+        pageIndexField.val ("" + (pageNumber - 1));
+        pageIndexField.trigger('change');
+        return true;
+      }
+    });
+  
+    pageNumberField.bind('keydown', function (event) {
+      var enterKey = 13;
+      var zeroKey = 48;
+      var nineKey = 57;
+      var zeroKeyNumBlock = 96;
+      var nineKeyNumBlock = 105;
+      var f1Key = 112;
+      var f12Key = 123;
+      var isEnterKey = event.keyCode == enterKey;
+      var isControlKey = event.keyCode < zeroKey || event.keyCode >= f1Key && event.keyCode <= f12Key;
+      var isNumericKey = event.keyCode >= zeroKey && event.keyCode <= nineKey || event.keyCode >= zeroKeyNumBlock && event.keyCode <= nineKeyNumBlock;
+  
+      if (isEnterKey)
+      {
+        pageNumberField.trigger("change");
+        event.cancelBubble = true;
+        event.stopPropagation();
+        return false;
+      }
+      else if (event.altKey || event.ctrlKey || isControlKey || isNumericKey)
+      {
+        return true;
+      }
+      else
+      {
+        return false;
+      }
+    });
+  }
+}
+
+function BocList_InitializeGlobals(): void
+{
+  BocList.InitializeGlobals();
+}
+
+function BocList_InitializeList(bocList: HTMLElement, selectRowSelectorControlName: string, selectAllSelectorControlName: string, selection: number, hasClickSensitiveRows: boolean, onSelectionChangedHandler: { (boclist: HTMLElement, flag: boolean): void }): void
+{
+  BocList.InitializeList(bocList, selectRowSelectorControlName, selectAllSelectorControlName, selection, hasClickSensitiveRows, onSelectionChangedHandler)
+}
+
+function BocList_OnSelectAllSelectorControlClick(bocList: HTMLElement, selectAllSelectorControl: JQuery, isRowHighlightingEnabled: boolean): void
+{
+  BocList.OnSelectAllSelectorControlClick(bocList, selectAllSelectorControl, isRowHighlightingEnabled)
+}
+
+function BocList_OnCommandClick(): void
+{
+  BocList.OnCommandClick()
+}
+
+function BocList_GetSelectionCount (bocListID: string): number
+{
+  return BocList.GetSelectionCount (bocListID)
+}
+
+function BocList_FixUpScrolling(bocList: JQuery): void
+{
+  BocList.FixUpScrolling(bocList);
+}
```

Comment:
Those two do not need to be public if I am not mistaken.

Reply:
`BocList_GetSelectionCount` is used in `BocList.cs` Line 3694, but you're right about `BocList_FixUpScrolling`